tests/flow/test_random_encode_decode.py

```diff
@@ -0,0 +1,40 @@
+from common import *
+from random_graph import create_random_schema, create_random_graph, run_random_graph_ops, ALL_OPS
+
+
+class test_random_encode_decode(FlowTestsBase):
+    def __init__(self):
+        self.env = Env(decodeResponses=True,
+                       moduleArgs='VKEY_MAX_ENTITY_COUNT 100 NODE_CREATION_BUFFER 100')
+        self.redis_con = self.env.getConnection()
+
+    def test01_random_graph(self):
```

Comment:
this file should be removed, the test should be added to `test_encode_decode.py`